src/dragonball/src/device_manager/mod.rs

```diff
@@ -714,6 +714,11 @@ impl DeviceManager {
 
         #[cfg(feature = "virtio-blk")]
         self.block_manager.remove_devices(&mut ctx)?;
+        #[cfg(feature = "virtio-net")]
```

Comment:
Could you add a FIXME here declearing that only after `dbs-virtio-devices`crate  is upgraded that this removal can be fully performed?

Reply:
I've added a fixme here.